src/Schema/Builder.php

```diff
@@ -134,21 +134,34 @@ public function drop($table)
         $blueprint->drop();
     }
 
-    /** @inheritdoc */
+/**
+ * @inheritdoc
+ *
+ * Drops the entire database instead of deleting each collection individually.
+ *
+ * In MongoDB, dropping the whole database is much faster than dropping collections
+ * one by one. The database will be automatically recreated when a new connection
+ * writes to it.
+ */
     public function dropAllTables()
     {
-        foreach ($this->getAllCollections() as $collection) {
```

Comment:
@GromNaN: This is now outside the scope of this PR, but I noted that `getAllCollections()` could more easily use [`Database::listCollectionNames()`](https://www.mongodb.com/docs/php-library/current/reference/method/MongoDBDatabase-listCollectionNames/).

That said, I'm not even sure that method is still necessary unless you're overriding a base method. It was protected, but this seemed like the only reference to it (in this file at least).

Reply:
I'm adding a deprecation notice.